src/creatures/creatures_definitions.hpp

```diff
@@ -733,8 +767,9 @@ enum CombatType_t : uint16_t {
 	COMBAT_ICEDAMAGE = 1 << 9,
 	COMBAT_HOLYDAMAGE = 1 << 10,
 	COMBAT_DEATHDAMAGE = 1 << 11,
+	COMBAT_NEUTRALDAMAGE = 1 << 12,
 
-	COMBAT_COUNT = 12
+	COMBAT_COUNT = 13
```

Comment:
`src/creatures/players/wheel/wheel_definitions.hpp` has "12" which is a reference to this, I am not sure why we can't use `COMBAT_COUNT` there, but this here breaks that now that it's 13 and not 12